yarn-project/blob-lib/src/blob.test.ts

```diff
@@ -104,39 +104,36 @@ describe('blob', () => {
     expect(isValid).toBe(true);
   });
 
-  it('should evaluate full blobs', async () => {
+  it('should evaluate full blob', async () => {
     // This test ensures that the Blob class correctly matches the c-kzg lib
-    // The values here are used to test Noir's blob evaluation in noir-projects/noir-protocol-circuits/crates/blob/src/blob.nr -> test_full_blobs
+    // The values here are used to test Noir's blob evaluation in noir-projects/noir-protocol-circuits/crates/blob/src/blob.nr -> test_full_blob
```

Comment:
Modified this test to evaluate a single blob, since we test multiple blobs and their batching plenty in other places now.